deploy_stack.sh

```diff
@@ -5,6 +5,28 @@ if ! [ -x "$(command -v docker)" ]; then
   exit 1
 fi
 
-echo "Deploying stack"
-docker stack deploy func --compose-file docker-compose.yml
+export ARCH=$(uname -m)
+
+case "$ARCH" in
+
+"armv7l" | "armv6l" )            COMPOSEFILE="docker-compose.armhf.yml"
+                                 ;;
+
+"x86_64")                        COMPOSEFILE="docker-compose.yml"
+                                 ;;
+
+"aarch64" | "armv8l")           COMPOSEFILE="docker-compose.arm64.yml"
+                                 ;;
+
+*) echo "Sorry, your architecture ($ARCH) cannot currently be matched to a compose file."
+   exit 1
+   ;;
+esac
+
+if [ "$ARCH" = "x86_64" ] && [ "$1" = "extended" ] ; then
+COMPOSEFILE="docker-compose.extended.yml"
```

Comment:
Is there no extended compose file for arm yet? We should look at that because NATS has support as a multi-arch image now.

Reply:
#322 logged to look at extended compose file(s) for arm